README.md

```diff
@@ -330,11 +334,17 @@ but they can also be [configured for personal repositories](https://docs.github.
 
 These secrets can also be found at the top of [flowzone.yml](./.github/workflows/flowzone.yml).
 
+#### `PRIVATE_KEY`
```

Comment:
I'd prefer a more specific name than `PRIVATE_KEY`, eg `GH_APP_PRIVATE_KEY`
```suggestion
#### `GH_APP_PRIVATE_KEY`
```
or maybe adding the ephemeral bit in there as well could make sense?